app/models/file_set.rb

```diff
@@ -2,5 +2,13 @@
 
 # Generated by hyrax:models:install
 class FileSet < ActiveFedora::Base
+  property :is_derived,
```

Comment:
Is this added to the valkyrie models as well?

Reply:
Probably needs to be added…boy howdy I don't know about updating `Hyrax::FileSet`